packages/website/src/bin/page-generator/write-pages.integration.test.js

```diff
@@ -37,6 +37,18 @@ describe('Page templates', () => {
     assertNoAbsoluteImports(output);
   });
 
+  it('creates js for a package home page even when there is no README file', () => {
+    generators.generateHomePage(
+      'output.js',
+      '',
+      {},
+      { wrappersPath, pagesPath },
+    );
+
+    const output = getOutput('output.js');
+    expect(output).not.toMatch(/^import .+ from '\undefined'/m);
```

Comment:
I believe you can also do `new RegExp(`^import .+ from 'undefined'`, 'm')` to avoid escaping